_pages/documentation/general_docs/gpu_models/amdgpu.md

```diff
@@ -77,22 +81,26 @@ The following additional components are used to build and run machine learning p
 * [rocBLAS](https://github.com/ROCmSoftwarePlatform/rocBLAS/)
 * [MIOpen](https://github.com/ROCmSoftwarePlatform/MIOpen/)
 * [rocm-cmake](https://github.com/RadeonOpenCompute/rocm-cmake/)
+* [PyTorch](https://pytorch.org/) (FS mode only)
+* [Tensorflow](https://www.tensorflow.org/) - specifically the tensorflow-rocm python package (FS mode only)
 
 For information about installing these components locally, the commands in the GCN3 dockerfile (`util/dockerfiles/gcn-gpu/`) can be followed on an Ubuntu 16 machine.
 
 ## **Documentation and Tutorials**
 
-#### GCN3 Model
-Describes the GCN3 model
+Note that the VEGA ISA is a newer, superset ISA derived from GCN3. Therefore, the contents of the following papers, tutorials, and documentation apply to VEGA as well.
+
+#### GPUhttps://ieeexplore.ieee.org/document/8327041 Model
```

Comment:
Looks like the IEEE link was accidentally added in the middle of this line?

Reply:
Oops.. Will fix this